include/evmmax/evmmax.hpp

```diff
@@ -8,6 +8,17 @@
 namespace evmmax
 {
 
+namespace
```

Comment:
You shouldn't use anonymous namespace in a header—you will get a new copy of this function in every compilation unit.